tools/firmware.toit

```diff
@@ -123,6 +124,7 @@ main arguments/List:
       ]
   root_cmd.add create_cmd
   root_cmd.add extract_cmd
+  root_cmd.add esptool_cmd
```

Comment:
Should we put this under a subcommand so it is easier to extend for non-ESP targets in the future?

`firmware tool esptool`

Reply:
done.